plugins/trace_api_plugin/include/eosio/trace_api/trace.hpp

```diff
@@ -36,6 +47,21 @@ namespace eosio { namespace trace_api {
      chain::transaction_header                  trx_header = {};
   };
 
+  struct transaction_trace_v2 {
```

Comment:
This shouldn't exist and can be added to v3.

Reply:
And v3 would become v2.

Reply:
And v3 would become v2. Done.